src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const action = Object.values(actions);
+
+  for (const key of action) {
```

Comment:
```suggestion
  for (const action of actions) {
```

Reply:
not fixed 
![image](https://user-images.githubusercontent.com/122989585/235303031-172c4d94-b362-4060-af00-c8b940b4c4eb.png)